smsdk/client.py

```diff
@@ -521,6 +578,30 @@ def get_line_data(
         :param limit: A limit of records to grab defaults to 400
         :param offset: The offset to start the data at
         """
+        assets = kwargs.pop("assets", None)
+        if assets is None and args:
+            assets = args[0]
+            args = args[1:]
+
+        fields = kwargs.pop("fields", None)
+        if fields is None and args:
+            fields = args[0]
+            args = args[1:]
+        elif fields is None:
+            fields = []
+
+        time_selection = kwargs.pop("time_selection", None)
+        if time_selection is None and args:
+            time_selection = args[0]
+            args = args[1:]
+        elif time_selection is None:
+            time_selection = ONE_DAY_RELATIVE
+
+        asset_time_offset = kwargs.pop("asset_time_offset", {})
+        filters = kwargs.pop("filters", [])
```

Comment:
I can't pass filters, limit or offset via args with this function

Reply:
Fixed it.